assets/uswds/js/uswds.js

```diff
@@ -5070,7 +5070,7 @@ object-assign
         const NAV_LINKS = `${NAV} a`;
         const NON_NAV_HIDDEN_ATTRIBUTE = `data-nav-hidden`;
         const OPENERS = `.${PREFIX}-menu-btn`;
-        const CLOSE_BUTTON = `.${PREFIX}-nav__close`;
+        const CLOSE_BUTTON = `.${PREFIX}-nav-close`;
```

Comment:
Is it possible to solve this without touching this file?
assets/uswds/js/uswds.js is from a library rather than one of the custom .js files from ARS or NAL.

Perhaps changing the class name to match this?
Or alternatively adding this code to another .js file that takes higher precedence?